sdk/security/keyvault/azsecrets/autorest.md

```diff
@@ -17,6 +17,17 @@ security-scopes: "https://vault.azure.net/.default"
 use: "@autorest/go@4.0.0-preview.46"
 version: "^3.0.0"
 
+declare-directive:
+  rename-property: >-
```

Comment:
Please add a comment on this one, I can't tell what it does 😆

Reply:
removed in [da9a800](https://github.com/Azure/azure-sdk-for-go/pull/20861/commits/da9a80082c721308944899e04818ff7ad034a6fd)